server/runtime_test.go

```diff
@@ -100,7 +100,9 @@ func runtimeWithModulesWithData(t *testing.T, modules map[string]string) (*Runti
 		leaderboardRankCache: lbRankCache,
 	}
 
-	rt, rtInfo, err := NewRuntime(ctx, logger, logger, db, protojsonMarshaler, protojsonUnmarshaler, cfg, "", nil, lbCache, lbRankCache, lbSched, nil, nil, nil, nil, nil, metrics, nil, &DummyMessageRouter{}, storageIdx)
+	tracker := &LocalTracker{}
```

Comment:
unrelated?

Reply:
tests were broken due to some changes that were introduced in another PR, unrelated but fixing them here